pyfftw/pyfftw.pyx

```diff
@@ -1354,6 +1357,14 @@ cdef class FFTW:
         which the FFT is taken. If the direction is ``'FFTW_FORWARD'``, this
         flag makes no difference to the output array.
 
+        If ``ortho`` is ``True``, then the output of both forward
+        and inverse DFT operations is scaled by 1/sqrt(N), where N is the
+        product of the lengths of input array on which the FFT is taken.  If
+        ortho is True then the 2-norm of fft(A) will equal the 2-norm of A.
```

Comment:
A little extra info would be helpful here. Something like: `i.e. energy is preserved across the transform`.